gslib/metrics.py

```diff
@@ -728,19 +728,25 @@ def ReportMetrics(
     if log_file_path is not None:
       # If the path is not None, we'll need to surround the path with quotes
       # so that the path is passed as a string to the metrics_reporter module.
-      log_file_path = '"%s"' % log_file_path
-
-    reporting_code = ('from gslib.metrics_reporter import ReportMetrics; '
-                      'ReportMetrics("{0}", {1}, log_file_path={2})').format(
-                          temp_metrics_file.name,
-                          log_level,
-                          log_file_path).encode('utf-8').decode('unicode_escape')
+      log_file_path = 'r"%s"' % log_file_path
+
+    reporting_code = six.ensure_str(
+        'from gslib.metrics_reporter import ReportMetrics; '
+        'ReportMetrics(r"{0}", {1}, log_file_path={2})'.format(
+            temp_metrics_file.name,
+            log_level,
+            log_file_path))
     execution_args = [sys.executable, '-c', reporting_code]
     exec_env = os.environ.copy()
     exec_env['PYTHONPATH'] = os.pathsep.join(sys.path)
-
+    sm_env = dict()
```

Comment:
What is "sm" abbreviating here?

Reply:
sub-module_environment

Reply:
Might be worth typing the extra characters, then. I wouldn't have guessed that one :)